packages/DropdownMenu/src/DropdownMenu.js

```diff
@@ -80,6 +76,8 @@ const DropdownMenu = props => {
       const confirmationComponent = renderConfirmation(handleCloseMenu);
       return React.cloneElement(confirmationComponent, {
         align,
+        edge,
```

Comment:
ensuring the confirmation component uses the same edge value as the rest of the items in the dropdown